inc/planningevent.class.php

```diff
@@ -595,10 +596,11 @@ static function populatePlanning($options = []) :array {
     * @param $type       position of the item in the time block (in, through, begin or end)
     *                    default '')
     * @param $complete   complete display (more details) (default 0)
+    * @param $options         allow to have planning typeview (array)
```

Comment:
what can contains `$options` array ?

Reply:
i used only $viewname instead of array options